.travis.yml

```diff
@@ -49,20 +49,20 @@ jobs:
         - cargo clean --doc && cargo doc --features secp256k1 --no-deps && cargo deadlinks --dir target/doc
 
     - stage: deploy
-      rust: nightly-2019-06-16
+      rust: nightly-2020-01-05
       script:
         - |
           cargo clean --doc && \
           cargo rustdoc --features secp256k1 -- -Z unstable-options \
-            --extern-html-root-url base64=https://docs.rs/base64/0.10.1 \
-            --extern-html-root-url exonum-crypto=https://docs.rs/exonum-crypto/0.11.0 \
-            --extern-html-root-url failure=https://docs.rs/failure/0.1.5 \
-            --extern-html-root-url digest=https://docs.rs/digest/0.8.0 \
-            --extern-html-root-url hmac=https://docs.rs/hmac/0.7.0 \
-            --extern-html-root-url crypto-mac=https://docs.rs/crypto-mac/0.7.0 \
-            --extern-html-root-url secp256k1=https://docs.rs/secp256k1/0.12.0 \
-            --extern-html-root-url serde_json=https://docs.rs/serde_json/1.0.39 \
-            --extern-html-root-url serde_cbor=https://docs.rs/serde_cbor/0.9.0
+            --extern-html-root-url base64=https://docs.rs/base64 \
```

Comment:
It seems that removing crate versions breaks links in the generated docs; at least this is what happens when I run this command locally. For example, a link to `base64::DecodeError` in `enum.ParseError.html#variant.Base64` becomes `https://docs.rs/base64/base64/decode/enum.DecodeError.html`, which is not a valid link (there should be a version or version requirement between two `base64` mentions in the URL). Could you restore versions (updating them when necessary)?

Reply:
fixed using docs.rs semver urls